.vscode/settings.json

```diff
@@ -107,9 +107,15 @@
         "csignal": "cpp",
         "format": "cpp",
         "*.in": "cpp",
-        "texture": "cpp"
+        "*.py.in": "python",
+        "texture": "cpp",
+        "charconv": "cpp",
+        "numbers": "cpp",
+        "scoped_allocator": "cpp",
+        "semaphore": "cpp"
     },
-    "C_Cpp.configurationWarnings": "Disabled",
+    "jupyter.notebookFileRoot": "${workspaceFolder}/build/python/install/lib/python3/dist-packages",
```

Comment:
@prudhomm @vincentchabannes @alielarif I'm not sure this line should stay here : if the preset selected is not `python` there is no reason this path should be correct.
The advantage of this setting it that when we run a notebook in vscode, it automatically find the development packages installed in `build` directory, so there is no need to set by hand `$PYTHONPATH`.
The drawback is that when we launch a kernel, the working directory is no longer the directory where the ipynb file is saved, it is the one set in this line. So it is messy to load the config file (for example in p[python/pyfeelpp-mor/feelpp/mor/nirb/test_nirb_offline.ipynb](https://github.com/feelpp/feelpp/blob/e82a120d93c2791083f631431752c95031d6cd60/python/pyfeelpp-mor/feelpp/mor/nirb/test_nirb_offline.ipynb), the path is hard coded to a local path `"/home/saigre/Documents/code/feelpp/python/pyfeelpp-mor/feelpp/mor/nirb/model/square/square.cfg"` which should have not been pushed, my bad !)

I don't know what would be the best way to set the environment easily : automatically set `PYTHONPATH` to the install directory, while the working directory still being the one where the notebook file is saved...

Reply:
could you set this in your user settings and not the workspace setting ?